core/handlers/v2/schema.json

```diff
@@ -43,7 +43,24 @@
 				"matcher": {
 					"type": "string"
 				},
-				"value": {}
+				"value": {},
+				"config": {
+					"properties": {
+						"ignoreUnknown": {
```

Comment:
We need to change documentation as well in this file - docs/pages/reference/hoverfly/request_matchers.rst